DVCodingStyle.md

```diff
@@ -1074,7 +1074,10 @@ assert(function_that_has_side_effects());
 Always put `wait fork` and `disable fork` constructs inside of an
 `isolation fork...join` block to avoid erroneous waiting.
 It is recommended to use the `DV_SPINWAIT` macro whenever possible,
-[as described here](#macro-usage)
+[as described here](#macro-usage).
+
+Only `disable fork` may be used to terminate all subprocesses of the calling processes;
```

Comment:
This is incorrect. `disable label` is a perfectly valid way to stop the execution of the code block. I have explained in the Ibex bug what is actually wrong in that code. 

```systemverilog
  fork: label
    begin: thread1
    end
    ...
   join_any
  disable label; // INCORRECT
  disable thread1; // Perfectly valid, but does not respect parent child process relationships
  disable fork; // Kills all threads spawned by the parent thread, including the ones in the past
```

Depending on the situation, `disable thread1` and `disable fork`, both can be used. The above section clarifies the pitfall of `disable fork` and the need for an isolation fork. If `thread1` does not spawn more child threads, then `disable thread1` will obviate the need for the isolation fork which is required with `disable fork`. 

Please update this to say `disable label` is invalid only if `label` is the label given to the `fork` itself rather than the child threads. Also, indicate the pitfall of `disable thread1`.

Reply:
Sri, could you please suggest a specific wording to add to avoid going back and forth on this? Thanks! (Note that this text is in the "`"Wait and Fork" section.)

Reply:
I've proposed a solution here, which hopefully complies.